java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/util/StringLineAppender.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.runtimes.builder.util;
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Utility class for building strings comprised of multiple lines.
+ */
+public class StringLineAppender {
```

Comment:
I don't see all that much utility over a `List<String>` but I guess it's a matter of style. :smile: